website/docs/d/vapp_vm.html.markdown

```diff
@@ -71,6 +71,16 @@ The following arguments are supported:
 * `vdc` - (Optional) The name of VDC to use, optional if defined at provider level
 * `vapp_name` - (Required) The vApp this VM belongs to.
 * `name` - (Required) A name for the VM, unique within the vApp 
+* `network_dhcp_wait_seconds` - (Optional; *v2.7+*) Allows to wait for up to a defined amount of
+  seconds before IP address is reported for NICs with `ip_allocation_mode=DHCP` setting. It
+  constantly checks if IP is reported so the time given is a maximum. VM must be powered on and 
+  __at least one__ of the folowing __must be true__:
+ * VM has guest tools. It waits for IP address to be reported in vCD UI. This is a slower option, but
```

Comment:
> waits for IP address to be reported in vCD UI

The mention of the UI here and below are a bit unexpected. I'm pretty sure you are not waiting for vCD UI - you're checking the vCD API. We should change the formulation in such a way, that user is not left thinking that we are querying the UI (which we are not). I mean, we can mention that "at that point, UI reports IP address as well" or something like that, but we should not make it seem like provider is depended on UI.